src/tests/suite/daos_nvme_recovery.c

```diff
@@ -115,7 +115,7 @@ nvme_recov_1(void **state)
 	 *	  is ready.
 	 */
 	print_message("Waiting for faulty reaction done...\n");
```

Comment:
may print msg that will wait 180 second... or people may think it looks like hang there